Neos.ContentRepository/Classes/Domain/Repository/NodeDataRepository.php

```diff
@@ -1058,8 +1058,9 @@ public function addNodeTypeFilterConstraintsToQueryBuilder(QueryBuilder $queryBu
      * @param string $nodeTypeFilter
      * @return array
      */
-    protected function getNodeTypeFilterConstraintsForDql($nodeTypeFilter)
+    protected function getNodeTypeFilterConstraintsForDql($nodeTypeFilter = '')
```

Comment:
So this fallback value for the argument is never really used in PHP7? Strange (new) behaviour